.venv/project/ecommerce/views.py

```diff
@@ -1,4 +1,47 @@
+import uuid
 from django.http import HttpResponse
+from django.shortcuts import render,redirect
+
+# Account Creation/Login Forms Imports
+from .controllers.SignupForm import BuyerSignupForm 
+
+# Firestore Imports
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import firestore
+from firebase_admin import auth
+
+
 
 def home(request):
-    return HttpResponse("Hello, SOEN 341 Group!")
+    return render(request,'home.html')
+    #return HttpResponse("Hello, SOEN 341 Group!")
+
+def login(request):
+    return render(request, 'login.html')
+
+def signup(request):
```

Comment:
This code should be refactored to a centralized api for these calls. It will be better to organize calls to the database in a centralized location so that we don't have this type of code all over the place.